labs/RivePlayer/samples/RivePlayer.Samples/RivePlayerCustomSample.xaml

```diff
@@ -0,0 +1,67 @@
+﻿<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="RivePlayerExperiment.Samples.RivePlayerCustomSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:i="using:Microsoft.Xaml.Interactivity"
+      xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
+      xmlns:local="using:RivePlayerExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:rive="using:CommunityToolkit.Labs.WinUI.Rive"
+      mc:Ignorable="d">
+
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="400" />
+        </Grid.ColumnDefinitions>
+        <rive:RivePlayer x:Name="RivePlayer"
+                         Grid.Column="0"
+                         Width="600"
+                         Height="600"
+                         DrawInBackground="True"
+                         Source="ms-appx:///animated-login-screen.riv">
+            <rive:BoolInput Target="isChecking"
+                            Value="{x:Bind IsChecking.IsChecked, Mode=OneWay}" />
+            <rive:NumberInput Target="numLook"
+                              Value="{x:Bind NumLook.Value, Mode=OneWay}" />
+            <rive:BoolInput Target="isHandsUp"
+                            Value="{x:Bind IsHandsUp.IsChecked, Mode=OneWay}" />
+            <rive:TriggerInput x:Name="FailureTrigger"
+                               Target="trigFail" />
+            <rive:TriggerInput x:Name="SuccessTrigger"
+                               Target="trigSuccess" />
+        </rive:RivePlayer>
+        <StackPanel Grid.Column="1"
```

Comment:
We should move these to the SampleOptions. but I don't think we support all of these types of options yet. So something to fix later on :).